include/openmc/dagmc.h

```diff
@@ -9,11 +9,15 @@ extern "C" const bool dagmc_enabled;
 #ifdef DAGMC
 
 #include "DagMC.hpp"
-#include "openmc/cell.h"
-#include "openmc/surface.h"
+#include "openmc/xml_interface.h"
+#include "openmc/position.h"
 
 namespace openmc {
 
+extern moab::DagMC::RayHistory history;
+extern Direction last_dir;
+#pragma omp threadprivate(history, last_dir)
```

Comment:
Does this work on the intel compiler?

Reply:
It does. I think because they aren't templated types?

Reply:
right, makes sense